package.json

```diff
@@ -71,13 +71,12 @@
     "@typescript-eslint/eslint-plugin": ">=4.0.1"
   },
   "devDependencies": {
-    "@commitlint/cli": "11.0.0",
```

Comment:
Don't update the version unless you have to, for the purpose of this PR (which is about replacing `commitlint-config-awesome` with `@commitlint/config-conventional`.

Reply:
I don't think it is viable to open another PR just to update a version, it is still related to this PR even if it is indeed not necessary or for the purpose of this PR.

Reply:
I disagree. Related is not enough. PRs in this project are cheap. If you don't feel like making another PR—don't. But please don't include it with this one.

Reply:
Right, it's up to you! I will open a new PR if needed! :smile: